momaland/envs/connect4/connect4.py

```diff
@@ -299,43 +299,52 @@ def render(self):  # TODO adapt to different board sizes, low priority
             if self.render_mode == "human":
                 pygame.display.set_caption("Connect Four")
                 self.screen = pygame.display.set_mode((screen_width, screen_height))
-            elif self.render_mode == "rgb_array":
+            else:
                 self.screen = pygame.Surface((screen_width, screen_height))
 
         # Load and scale all of the necessary images
-        tile_size = (screen_width * (91 / 99)) / 7
+        size_cap = min(screen_width, screen_height)
+        tile_size = (size_cap * 91 / 99) / self.board_width
 
         red_chip = get_image(os.path.join("img", "C4RedPiece.png"))
         red_chip = pygame.transform.scale(red_chip, (int(tile_size * (9 / 13)), int(tile_size * (9 / 13))))
 
         black_chip = get_image(os.path.join("img", "C4BlackPiece.png"))
         black_chip = pygame.transform.scale(black_chip, (int(tile_size * (9 / 13)), int(tile_size * (9 / 13))))
 
-        board_img = get_image(os.path.join("img", "Connect4Board.png"))
-        board_img = pygame.transform.scale(board_img, ((int(screen_width)), int(screen_height)))
+        cell_img = get_image(os.path.join("img", "c4-empty.png"))
+        cell_img = pygame.transform.scale(cell_img, (int(tile_size), int(tile_size)))
 
-        self.screen.blit(board_img, (0, 0))

```

Comment:
The window resizing seems to work only for changed board width, not for changed board height (e.g. with a board height of >6 cells)

Reply:
I fixed this issue now